application-analytics-ui/src/main/resources/Analytics/Code/AnalyticsTranslations.xml

```diff
@@ -282,6 +282,24 @@ analytics.visitsOverview.visits.option=Visits
 rendering.macro.visitsOverview.description=This is an overview of the visit evolution.
 rendering.macro.visitsOverview.name=Visits overview
 
+## ReturningVisitsOverTime Macro
+analytics.returningVisitsOverTime.avgTimeOnSiteNew.option=Average visit duration for new visitors
+analytics.returningVisitsOverTime.bounceRateNew.option=New visits have bounced(left the website after one page)%
+analytics.returningVisitsOverTime.nbActionsByTheReturningVisits.option=Actions by the returning visits
+analytics.returningVisitsOverTime.nbActionsNew.option=Actions by the new visits
+analytics.returningVisitsOverTime.nbActionsPerVisitNew.option=Actions per new visit
+analytics.returningVisitsOverTime.nbActionsPerVisitReturning.option=Actions per returning visit.
```

Comment:
This is ending with `.`, while the others are not, and this is strange in the UI